lib/compress/zstd_compress_internal.h

```diff
@@ -192,6 +192,13 @@ typedef struct {
   size_t capacity; /* The capacity starting from `seq` pointer */
 } rawSeqStore_t;
 
+typedef struct {
+    int collectSequences;
+    ZSTD_Sequence* seqStart;
+    ZSTD_Sequence* seqCurrent;
```

Comment:
minor: API style:

we have something loosely equivalent in the streaming interface : [`ZSTD_outBuffer`](https://github.com/facebook/zstd/blob/dev/lib/zstd.h#L555).

The general idea is that it's a bit better to represent a limited and partially filled destination buffer as `ptr, usize, usize` rather than `ptr, ptr, usize`.
One reason is that it's easier to reason about bounds, hence to detect and reject incorrect inputs.

Applied here, this would result in : 
```C
   ZSTD_Sequence* seqStart;
   size_t nbSeqs;
   size_t scCapacity;
```